build/mixxx.py

```diff
@@ -39,8 +39,8 @@ def __init__(self, target, machine, build, toolchain, available_features=[]):
         if build is None:
             build = 'debug'
 
-        if not build in ['debug', 'release']:
-            raise Exception("invalid build type")
+        if not build in ['debug', 'release', 'none']:
```

Comment:
Yes, I realize why the change was made. I'm saying I don't think it's the right way to accomplish the desire of a no-flags build. The code is more complicated if we have to consider build 3 build types. There's no "none" MSVC runtime -- the user needs to pick release or debug.